filebeat/crawler/prospector_log.go

```diff
@@ -228,6 +228,8 @@ func (p *ProspectorLog) checkExistingFile(h *harvester.Harvester, newFile *input
 
 			// Start a new harvester on the path
 			h.Start()
+			p.Prospector.registrar.Persist <- h.GetState()
```

Comment:
while unlikely I always see a race between harvester and prospector in this construct:

1. start harvester
2. have prospector get harvester state (running in parallel)
3. send 'current' harvester state to registrar.